source/items/siftermeshes.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Sifter Meshes
+layout: default
+---
+# Sifter Meshes
+
+<div class="infobox box text-center">
+    <p style="text-align:center;"><img src="../../assets/images/items/mesh_string.png" alt="String Sifter Mesh"></p>
+    <p style="text-align:center;"><img src="../../assets/images/items/mesh_flint.png" alt="Flint Sifter Mesh"></p>
+    <p style="text-align:center;"><img src="../../assets/images/items/mesh_iron.png" alt="Iron Sifter Mesh"></p>
+    <p style="text-align:center;"><img src="../../assets/images/items/mesh_diamond.png" alt="Diamond Sifter Mesh"></p>
+    Such a variety of materials requires a variety of specialized crafters, and the knowledge to use them.
+</div>
+<br>
+
+MineColonies adds 4 Meshes, used by the [Sifter](../../source/workers/sifter). The Sifter will automatically unlock these recipes for your colony once the requirements are met. More advanced meshes last longer, and can sift out materials earlier meshes would have missed.
```

Comment:
```suggestion
MineColonies adds four meshes used by the [Sifter](../../source/workers/sifter) to sift through items. The Sifter will automatically unlock these recipes for your colony once the requirements are met. More advanced meshes last longer and can sift out materials earlier meshes would have missed.
```

Reply:
also maybe add a note about the progression of the meshes (string, flint, iron, diamond)